### ISSUE TITLE
Is generation of two files normal?

### ISSUE THREAD

**Issue by Sevrex** [KhronosGroup/glTF-Blender-Exporter#229]
When I use the exporter I get model.gltf and buffer_model.gltf, s this normal behavior and if so is it possible to just have a single file? I am creating an online gallery tool and I want to encourage users to upload in glTF format but it will be a much better experience if they can upload a single file per model.

**Comment by emackey** [comment]
Yes, the `.glb` export is the "binary" glTF format, where everything is rolled into a single file, including texture images.

The `.gltf` file is a text-only file that references external assets, such as a `.bin` file for mesh data and image files for textures.

**Comment by donmccurdy** [comment]
For an online gallery tool, I would suggest only accepting the `.glb` (single-file) variant. This is what Facebook does, for example.
